samples/azure/performance-tuning-asb/ASB_7/FastSender/packages.config

```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net452" />
```

Comment:
Did you pick the wrong side of comparison. Seems to indicate a version down grade?